includes/class-activitypub.php

```diff
@@ -585,10 +587,8 @@ public static function default_actor_extra_fields( $extra_fields, $user_id ) {
 				'post_status'    => 'publish',
 				'post_author'    => $user_id,
 				'post_content'   => sprintf(
-					'<!-- wp:paragraph --><p><a rel="me" title="%s" target="_blank" href="%s">%s</a></p><!-- /wp:paragraph -->',
-					\esc_attr( $url ),
-					$url,
-					\wp_parse_url( $url, \PHP_URL_HOST )
+					'<!-- wp:paragraph --><p>%s</p><!-- /wp:paragraph -->',
+					Urls::the_content( $url )
```

Comment:
Is it even possible to post a text-url that will not be transformed to a link from the block editor?

Reply:
In that case the URLS comes from the author or blog. They should be convertred without problems.

Reply:
Sidenot  the `<p>` tags in Mastodon are inside of a `<span>`.

Reply:
and `<p>` inside of `<span>` is not valid in terms of HTML5

Reply:
Exactly